src/kokkos/plugin-PixelTriplets/kokkos/gpuPixelDoublets.h

```diff
@@ -3,8 +3,10 @@
 
 #include "gpuPixelDoubletsAlgos.h"
 
-#ifdef TODO
+#ifdef KOKKOS_BACKEND_CUDA
 #define CONSTANT_VAR __constant__
+#else
+#define CONSTANT_VAR
```

Comment:
Did you try creating a functor class with the constants as member data? (I vaguely recall we discussed that option)

(I'm also fine if we try it later)

Reply:
will do.